packages/beacon-node/src/util/blobs.ts

```diff
@@ -56,40 +63,46 @@ export function computeBlobSidecars(
   });
 }
 
-export function computeDataColumnSidecars(
+/**
+ * Turns a SignedBeaconBlock and an array of Blobs from a given slot into an array of
+ * DataColumnSidecars that are ready to be served by gossip and req/resp.
+ *
+ * Implementation of get_data_column_sidecars
+ * https://github.com/ethereum/consensus-specs/blob/dev/specs/_features/eip7594/das-core.md#get_data_column_sidecars
+ */
+export function getDataColumnSidecars(
   config: ChainForkConfig,
   signedBlock: SignedBeaconBlock,
   contents: deneb.Contents & {kzgCommitmentsInclusionProof?: electra.KzgCommitmentsInclusionProof}
 ): electra.DataColumnSidecars {
   const blobKzgCommitments = (signedBlock as deneb.SignedBeaconBlock).message.body.blobKzgCommitments;
+  const {blobs} = contents;
+  if (!Array.isArray(blobs)) {
```

Comment:
based on the type this shouldn't happen?

Reply:
Looking back at the calling context you are correct. Updated